scripts/v3.12/php-7.4/secp256k1/APKBUILD

```diff
@@ -7,10 +7,10 @@ pkgname=libsecp256k1
 _pkgname=secp256k1
 srcdir="/tmp/src-$_pkgname"
 pkgbasedir="/tmp/pkg-$pkgname"
-pkgver=20201211
+pkgver=20201209
 _soname=0.0.0
 pkgrel=2
-pkggitrev=95e99f196fd08a8b2c236ab99d7e7fec8f6dc78f
+pkggitrev=b61f9da54eff6d8137e0681b403b48be62f0460a
```

Comment:
Please revert to `95e99f196fd08a8b2c236ab99d7e7fec8f6dc78f`. Any newer rev breaks the extension.